db/index.js

```diff
@@ -5,7 +5,7 @@ const Promise = require('bluebird');
 mongoose.Promise = Promise;
 
 // The default url only works on local machines
-let connectionURL = 'mongodb://localhost/greenfield';
+var connectionURL = 'mongodb://localhost/greenfield';
```

Comment:
Since they are both in the outermost scope of their file, how is the scope any different?

Reply:
It does technically still work (you didn't destroy any functionality) so I can accept this part of your pull request, but I don't understand why you did it.